packages/ragbits-core/docs/how-to/sparse-vectors-with-vector-stores.md

```diff
@@ -0,0 +1,106 @@
+# How to Use Sparse Embeddings with Vector Stores
```

Comment:
You already added a doc on sparse embeddings in this very PR, so why add another one?

Besides, we keep our documentation in the top-level docs/ directory, not inside individual packages.